data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -413,8 +414,11 @@ then
         touch cb-bootstrap.done
         record_service_stage_success
     else
+        record_service_stage_failure
         ret=$?
         set +u
+	check_api_url
+	check_api_int_url
```

Comment:
In my testing, these functions are never being executed.

Reply:
The goal is to inform users when the bootstrap node can't connect to api-int. What i'm seeing is that with the current PR users will never see these error messages, because the way problems with resolving api-int manifest the relevant code would never be reached. 

So I think we need to move where/how these checks are being performed. Is there a good reason to only run them conditionally if cluster bootstrap fails?

Reply:
Running it conditionally when bootstrap fails was the recommendation provided in the JIRA story. But,  given the information after testing, it should be ok for us to move to a different location where it is useful. 
I'll incorporate that in my next patch.

Reply:
Yes. Note that "bootstrap" refers to the whole bootstrapping process, probably most closely associated with the bootkube service. What you have tied the check to in this PR is cluster bootstrap, which is just one part of the bootstrap process.

Reply:
The patch is looking at the Infrastructure.Status fields so it can't be too early in the bootstrap process (I think). I have now moved it so that it runs all the time after MCO has been started and before the REQUIRED_PODS have been started. The earlier location was after REQUIRED_PODS failed to come up.